lib/ansible/module_utils/urls.py

```diff
@@ -860,11 +926,12 @@ def open(self, method, url, data=None, headers=None, use_proxy=None,
                 client_cert=client_cert,
                 client_key=client_key,
             )
-        if unix_socket:
-            ssl_handler = UnixHTTPSHandler(unix_socket=unix_socket, context=context)
-        else:
-            ssl_handler = urllib.request.HTTPSHandler(context=context)
-        handlers.append(ssl_handler)
+        if HAS_SSL:
+            if unix_socket and UnixHTTPSHandler:
```

Comment:
If adding the `HAS_SSL` check, might as well simplify to:

```
if HAS_SSL and unix_socket and UnixHTTPSHandler:
```

Reply:
I've removed that branch. I'm not sure what I was stuck up on before, but I don't seem to be getting a mypy error any longer.